src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue

```diff
@@ -1,15 +1,129 @@
 <template>
-  <div>
-    Future page for Travel Estimates
-  </div>
+  <v-container fluid class="container-max-width">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">
+          How do you want to estimate your travel needs?
+        </h1>
+        <p>
```

Comment:
please add class `page-intro` to this `p` tag to reduce the space between header and text